src/ApiResourceDataTable.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Yajra\DataTables;
+
+use Illuminate\Support\Collection;
+use Illuminate\Pagination\LengthAwarePaginator;
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+
+class ApiResourceDataTable extends CollectionDataTable
+{
+    /**
+     * Collection object.
+     *
+     * @var Illuminate\Http\Resources\Json\AnonymousResourceCollection
+     */
+    public $collection;
+
+    /**
+     * Collection object.
+     *
+     * @var Illuminate\Http\Resources\Json\AnonymousResourceCollection
+     */
+    public $original;
+
+    /**
+     * Can the DataTable engine be created with these parameters.
+     *
+     * @param mixed $source
+     * @return bool
+     */
+    public static function canCreate($source)
+    {
+        return is_array($source) || $source instanceof AnonymousResourceCollection;
+    }
+
+    /**
+     * Factory method, create and return an instance for the DataTable engine.
+     *
+     * @param array|Illuminate\Http\Resources\Json\AnonymousResourceCollection $source
```

Comment:
@param should be `* @param \Illuminate\Http\Resources\Json\AnonymousResourceCollection $source`